Makefile

```diff
@@ -20,52 +20,28 @@ lint:
 	--rm tests golangci-lint run --exclude="(conf\.(Un)?[S,s]etEnv)" --deadline=$(LINT_TIMEOUT) --verbose
 	docker-compose -f docker-compose.test.yml run --rm tests gosec ./...
 
-# The following vars are available to tests needing SSAS admin credentials; currently they are used in smoke-test
-# Note that these variables should only be used for smoke tests, must be set before the api starts, and cannot be changed after the api starts
-SSAS_ADMIN_CLIENT_ID ?= 31e029ef-0e97-47f8-873c-0e8b7e7f99bf
-SSAS_ADMIN_CLIENT_SECRET := $(shell docker-compose run --rm ssas sh -c 'main --reset-secret --client-id=$(SSAS_ADMIN_CLIENT_ID)'|tail -n1)
-
-#
-# The following vars are used by both smoke-test and postman to pass credentials for obtaining an access token.
-# The CLIENT_ID and CLIENT_SECRET values can be overridden by environmental variables e.g.:
-#    export CLIENT_ID=1234; export CLIENT_SECRET=abcd; make postman env=local
-# or 
-#    CLIENT_ID=1234 CLIENT_SECRET=abcd make postman env=local
-#
-# If the values for CLIENT_ID and CLIENT_SECRET are not overridden, then by default, generate-client-credentials is
-# called using ACO CMS ID "A9994" (to generate credentials for the `ACO Dev` which has a CMS ID of A9994 in our test
-# data). This can be overridden using the same technique as above (exporting the env var and running make).
-# For example:
-#    export ACO_CMS_ID=A9999; make postman env=local
-# or
-#    ACO_CMS_ID=A9999 make postman env=local
-ACO_CMS_ID ?= A9994
-clientTemp := $(shell docker-compose run --rm api sh -c 'bcda reset-client-credentials --cms-id $(ACO_CMS_ID)'|tail -n2)
-CLIENT_ID ?= $(shell echo $(clientTemp) |awk '{print $$1}')
-CLIENT_SECRET ?= $(shell echo $(clientTemp) |awk '{print $$2}')
-
 smoke-test:
 	docker-compose -f docker-compose.test.yml build tests
-	BCDA_SSAS_CLIENT_ID=$(SSAS_ADMIN_CLIENT_ID) BCDA_SSAS_SECRET=$(SSAS_ADMIN_CLIENT_SECRET) test/smoke_test/smoke_test.sh
+	test/smoke_test/smoke_test.sh
 
 postman:
 	# This target should be executed by passing in an argument for the environment (dev/test/sbx)
 	# and if needed a token.
 	# Use env=local to bring up a local version of the app and test against it
 	# For example: make postman env=test token=<MY_TOKEN>
-	$(eval BLACKLIST_TEMP := $(shell docker-compose run --rm api sh -c 'bcda reset-client-credentials --cms-id A9997'|tail -n2))
-
-	$(eval BLACKLIST_CLIENT_ID:=$(shell echo $(BLACKLIST_TEMP) |awk '{print $$1}'))
-	$(eval BLACKLIST_CLIENT_SECRET:=$(shell echo $(BLACKLIST_TEMP) |awk '{print $$2}'))
+	$(eval BCDA_SSAS_CLIENT_ID=$(shell docker exec bcda-app_api_1 env | grep BCDA_SSAS_CLIENT_ID | cut -d'=' -f2))
```

Comment:
Need to get the credential and secret somehow without resetting them. So asking the container for it.